templates/vue/src/components/Lightbox/media/MultiContentMedia/PageMenu/PageMenuItem.vue

```diff
@@ -95,18 +96,22 @@ export default {
         !node.unlocked
       )
     },
-    handleTitleClick() {
-      this.scrollToRow()
+    nestedItemClick() {
+      this.itemClick()
     },
-    scrollToRow(nodeId) {
-      if (!nodeId) {
+    itemClick(nodeId) {

```

Comment:
It's better to name event handlers "handle..." so this is better called "handleItemClick"